db/migrate/20170629152622_add_deleter_to_escort.rb

```diff
@@ -0,0 +1,7 @@
+class AddDeleterToEscort < ActiveRecord::Migration[5.0]
```

Comment:
Shouldn't the name of the migration be something like "AddCancellerToEscort" or "AddCancellationColumnsToEscorts" ?